specs/latest/2.0/index.html

```diff
@@ -1073,15 +1167,17 @@ <h2 class="no-toc">Table of contents</h2>
         See <a href="#COPYING_BUFFERS">Copying Buffers</a> for restrictions imposed by the WebGL 2 API.
       </dd>
       <dt>
-        <p class="idl-code">void getBufferSubData(GLenum target, GLintptr offset, ArrayBuffer? returnedData)
+        <p class="idl-code">
+          void getBufferSubData(GLenum target, GLintptr offset,
+                                ByteBuffer returnedData,
+                                optional unsigned long byteOffset = 0)
         </p>
       </dt>
       <dd>
         For the WebGLBuffer bound to the passed target, read <code>returnedData.byteLength</code>
         bytes from the buffer starting at byte offset <code>offset</code> and write them
         to <code>returnedData</code>. If <code>offset + returnedData.byteLength</code> would extend
         beyond the end of the buffer an <code>INVALID_VALUE</code> error is generated.
-        If <code>returnedData</code> is null then an <code>INVALID_VALUE</code> error is generated.
```

Comment:
For this entry point let's leave the returnedData argument nullable so that it produces all of its errors in OpenGL style, rather than some in OpenGL style and some via DOM exceptions.